src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -138,11 +155,15 @@ public void MarkSimulationAsChanged()
          _simulation.HasChanged = true;
       }
 
-      public void RemoveOutputMapping(SimulationOutputMappingDTO outputMappingDTO)
+      public void RemoveObservedData(SimulationOutputMappingDTO outputMappingDTO)
       {
-         outputMappingDTO.Output = null;
-         _simulation.OutputMappings.Remove(outputMappingDTO.Mapping);
-         _view.RefreshGrid();
+         var usedObservedData = new UsedObservedData
+         {
+            Id = outputMappingDTO.ObservedData.Id,
```

Comment:
Why are we using an id here an not the ObservedData itself. This USedObservedData does not feel like the right structure to me (this might be used like this somewhere else but in this case..I don't get it. Later, you are going through the project to retrieve the observed data by id. all of that is not required if we just have a reference to the observed data?